agency/README.md

```diff
@@ -166,7 +166,11 @@ The route must include at least 2 points, a start point and end point. Additiona
 
 ## service_areas
 
-Gets the list of service areas available to the provider.
+Gets the list of service areas available to the provider. A service area is a rules engine that provides a set of service areas that implements [geofences and incentive areas](https://docs.google.com/document/d/170W8rWGMmBDb7U-4OvNAQgk_Jusk6UKrsE-BP-Gk5VQ/edit) along with a base layer set of rules for operation of devices inside a jurisdiction running the Agency endpoint. 
+
+The endpoint returns an a ordered list of GeoJSON MultiPolygons, per provider, shows which type of events are allowed in what areas. The list of the events in the same as defined in [provider/status_changes](https://github.com/CityOfLosAngeles/mobility-data-specification/tree/master/provider#status-changes). 
```

Comment:
The wording is a little garbled here.  How about
`The endpoint returns an ordered list of GeoJSON MultiPolygons, per provider, detailing which type of vehicle events are allowed in those areas.`